manual_user_install.sh

```diff
@@ -146,30 +153,20 @@ function install_extra_skills (){
     echo
     echo "Installing extra skills"
     echo
-    # from NeonGeckoCom
-    pip3 install git+https://github.com/NeonGeckoCom/skill-user_settings
-    pip3 install git+https://github.com/NeonGeckoCom/skill-spelling
-    pip3 install git+https://github.com/NeonGeckoCom/skill-local_music
-    pip3 install git+https://github.com/NeonGeckoCom/skill-caffeinewiz
-
-    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-weather
-    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-hello-world
 
-    # common query
-    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-ddg
-    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-wolfie
-    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-wikipedia
-
-    # fallback
-    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-fallback-chatgpt
+    INSTALL=""
+    # install the OVOS extra skills
+    for p in $OVOS_EXTRA_SKILLS_REPOS; do
+	INSTALL="$INSTALL ${OVOS_SOURCE}/$p"
+    done
 
-    # OCP
-    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-news
-    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-somafm
-    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-youtube-music
+    # Add the NeonGeckoCom skills
```

Comment:
I think I want to remove the Neon skills from the image.  Most are in OVOS repos, and the ones that aren't don't play well with OVOS all of the time.  And they bring in extra dependencies such as `neon-utils`

Reply:
I leave the Neon skills out of my builds... On a 3B+ I want to be as lightweight as possible. I didn't realize they were pulled from the image.